lib/verifier/hathifiles_redirects_verifier.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require "verifier"
+
+module PostZephirProcessing
+  class HathifileRedirectsVerifier < Verifier
```

Comment:
Class is `HathifileRedirectsVerifier` but file is named `hathifiles_redirects_verifier`. Which should we change?

Reply:
"Hathifiles" plural is nearer to being universal. However unfortunate, it's the least astonishing thing. I suggest using "hathifile" singular as a hash key or method parameter to refer to the path to or name of such a file.